src/commands/flow/csvColumns.ts

```diff
@@ -0,0 +1,9 @@
+export default {
```

Comment:
`csvColumns.ts` and `example.ts` should be located in `src/commands/flow/internal`

Reply:
fixed